### PR TITLE
targetting iOS 13.0 for swiftui

### PR DESCRIPTION
**Pull request by nhamming**

### CODE REVIEW

File: Loop.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3461,6 +3461,7 @@
 				CODE_SIGN_STYLE = "$(LOOP_CODE_SIGN_STYLE)";
 				DEVELOPMENT_TEAM = "$(LOOP_DEVELOPMENT_TEAM)";
 				INFOPLIST_FILE = Loop/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```
Comment (darinkrauss) [inline Loop.xcodeproj/project.pbxproj]: Please push this setting up to the project settings (not target specific).

**Review by darinkrauss — CHANGES_REQUESTED** [review]
Please push the target-specific changes up to the project level. Also, will need feedback from @ps2 about how this is going to play with DIY.

File: Loop.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nhamming) [reply Loop.xcodeproj/project.pbxproj]: Sure. I only updated the necessary targets, but I can update for the project instead.

**Review by darinkrauss — CHANGES_REQUESTED** [review]
Sorry, not to be pedantic, but I meant change the project settings (as you did) and *remove* the per-target settings (so just the project settings takes effect).

**Comment by darinkrauss** [comment]
My last big change to the Loop project (with the automated TestFlight build) made it so that all of the common settings were at the project level and only the few per-target specific changes were specified in the target settings.

**Comment by nhamming** [comment]
> Sorry, not to be pedantic, but I meant change the project settings (as you did) and _remove_ the per-target settings (so just the project settings takes effect).

My bad. Should have reverted.

**Review by darinkrauss — APPROVED** [review]
Thanks! LGTM! 🚀

**Pull request merged**